pkgs/by-name/za/zapret/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  libcap,
+  zlib,
+  libnetfilter_queue,
+  libnfnetlink,
+
+  iptables,
+  nftables,
+  gawk
+}:
+
+stdenv.mkDerivation {
+  pname = "zapret";
+  version = "0-unstable-2024-07-16";
+
+  src = fetchFromGitHub {
+    owner = "bol-van";
+    repo = "zapret";
+    rev = "9fcd8f830ebde2491719a5c698e22d1d5210e0fb";
+    sha256 = "sha256-8cqKCNYLLkZXlwrybKUPG6fLd7gmf8zV9tjWoTxAwIY=";
+  };
+
+  buildInputs = [ libcap zlib libnetfilter_queue libnfnetlink ];
+  nativeBuildInputs = [ iptables nftables gawk ];
+
+  buildPhase = ''
+    mkdir -p $out/bin
+
+    make TGT=$out/bin
+  '';
+
+  installPhase = ''
+    mkdir -p $out/usr/share/zapret/init.d/sysv
+    mkdir -p $out/usr/share/docs
+
+    cp $src/blockcheck.sh $out/bin/blockcheck
+
+    substituteInPlace $out/bin/blockcheck \
+      --replace "ZAPRET_BASE=\"\$EXEDIR\"" "ZAPRET_BASE=$out/usr/share/zapret"
+
+    cp $src/init.d/sysv/functions $out/usr/share/zapret/init.d/sysv/functions
+    cp $src/init.d/sysv/zapret $out/usr/share/zapret/init.d/sysv/init.d
+
+    substituteInPlace $out/usr/share/zapret/init.d/sysv/functions \
+      --replace "ZAPRET_BASE=\$(readlink -f \"\$EXEDIR/../..\")" "ZAPRET_BASE=$out/usr/share/zapret"
+
+    # i think the most of it can be overrided by env
+    cp $src/config $out/usr/share/zapret
```

Comment:
Isn't `config` sourced exactly to override the environment, though?

Reply:
Indeed, it is. I will remove config and sourcing config from source file